plugin/storage/memory/memory.go

```diff
@@ -339,3 +335,73 @@ func flattenTags(span *model.Span) model.KeyValues {
 	}
 	return retMe
 }
+
+func hasCorrespondingServerSpan(clientSpan *model.Span, spans []*model.Span) bool {
+	for _, span := range spans {
+		if span.ParentSpanID() == clientSpan.SpanID {
+			return true
+		}
+	}
+	return false
+}
+
+func isClientSpan(span *model.Span) bool {
+	for _, tag := range span.Tags {
```

Comment:
iirc `span.Tags.FindByKey()` function exists

Reply:
Updated the code block to use `span.Tags.FindByKey()`